### PR TITLE
Update all articles covering DevTools Device Mode

### PR DESCRIPTION
**Pull request by pbakaus**
Please take a look @kaycebasques!

### CODE REVIEW

**Comment by pbakaus** [comment]
@petele or @gauntface can you help me create a redirect that does the following along with this PR:

/web/tools/chrome-devtools/iterate/device-mode/media-queries (old)
=> /web/tools/chrome-devtools/iterate/device-mode/emulate-mobile-viewports (new)

**Comment by petele** [comment]
Once @kaycebasques has LGTM'ed, no problem. Was planning to do a push Thursday around 11am ET.

**Comment by kaycebasques** [comment]
@petele can you redirect /web/tools/chrome-devtools/iterate/inspect-styles/basics to /web/tools/chrome-devtools/iterate/inspect-styles too?

File: src/content/en/tools/chrome-devtools/index.markdown [hunk 1]
```diff
@@ -80,9 +80,8 @@ translation_priority: 1
         <p>Use the Device Mode to build fully responsive, mobile-first web experiences.</p>
         <ul>
           <li><a href="/web/tools/chrome-devtools/iterate/device-mode/">Device Mode</a></li>
-          <li><a href="/web/tools/chrome-devtools/iterate/device-mode/emulate-mobile-viewports">Emulate Mobile Viewports</a></li>
+          <li><a href="/web/tools/chrome-devtools/iterate/device-mode/emulate-mobile-viewports">Test Responsive and Device-specific Viewports</a></li>
           <li><a href="/web/tools/chrome-devtools/iterate/device-mode/device-input-and-sensors">Emulate Device Input and Sensors</a></li>
```
Comment (kaycebasques) [inline src/content/en/tools/chrome-devtools/index.markdown]: Needs to match the new title?

File: src/content/en/tools/chrome-devtools/iterate/device-mode/testing-other-browsers.markdown [hunk 2]
```diff
@@ -1,6 +1,6 @@
 ---
 layout: shared/narrow
-title: "Emulate beyond Chrome"
+title: "Emulate and Test Other Crowsers"
```
Comment (kaycebasques) [inline src/content/en/tools/chrome-devtools/iterate/device-mode/testing-other-browsers.markdown]: s/Crowsers/Browsers

**Comment by kaycebasques** [comment]
@pbakaus the old guidelines for annotating images say to use green for annotations. I'm cool with your use of blue, but going forward consistency would be good

File: src/content/en/tools/chrome-devtools/iterate/device-mode/index.markdown [hunk 3]
```diff
@@ -23,15 +25,11 @@ notes:
 * Evaluate your site's performance using the [network emulator](/web/tools/chrome-devtools/profile/network-performance/network-conditions), without affecting traffic to other tabs.
```
Comment (kaycebasques) [inline src/content/en/tools/chrome-devtools/iterate/device-mode/index.markdown]: The "inspecting CSS media queries" link above yields 404

File: src/content/en/tools/chrome-devtools/iterate/device-mode/index.markdown [hunk 4]
```diff
@@ -23,15 +25,11 @@ notes:
...
 * Accurately [simulate device input](/web/tools/chrome-devtools/iterate/device-mode/device-input-and-sensors) for touch events, geolocation, and device orientation
 
-Some of this documentation might be ahead of the stable version of Chrome. Use <a href="https://www.google.com/intl/en/chrome/browser/canary.html">Chrome Canary</a>.
+## How to disable Device Mode
 
-## How to enable Device Mode
-
-Turn on device mode by pressing the **Toggle device mode** ![toggle device mode icon off](imgs/icon-device-mode-off.png){:.inline} icon. When device mode is enabled, the icon turns blue and the viewport transforms into a device emulator.
+Device Mode is enabled by default. If you're building a Desktop-only site, you may turn it off by pressing the **Toggle device mode** ![toggle device mode icon on](imgs/device-mode-on.png){:.inline.wf-devtools-icon} icon. When Device Mode is disabled, the icon turns grey and the viewport controls are hidden.
```
Comment (kaycebasques) [inline src/content/en/tools/chrome-devtools/iterate/device-mode/index.markdown]: For consistency can you wrap the inline icon in parentheses? E.g. "by pressing the **Toggle device mode** () icon"

File: src/content/en/tools/chrome-devtools/iterate/device-mode/emulate-mobile-viewports.markdown [hunk 5]
```diff
@@ -1,122 +1,168 @@
...
-web design becomes all the more important. Web content needs to look and feel 
-great across a wide variety of devices and network conditions.</p>
-
-But testing the quality of your mobile experiences on a multitude of devices 
-takes longer and makes debugging more complex. Chrome DevTools' Device Mode 
-lets you mimic how your development site will look in production on a range of 
-devices.
-
-![device mode enabled](imgs/device-mode.png)
+<p class="intro">The updated Device Mode (since Chrome 49) is an integral part of the now-mobile-first DevTools and extend the main DevTools bar. Learn how to use its controls to simulate a wide range of devices or go fully responsive.</p>
```
Comment (kaycebasques) [inline src/content/en/tools/chrome-devtools/iterate/device-mode/emulate-mobile-viewports.markdown]: s/extend/extends

File: src/content/en/tools/chrome-devtools/iterate/device-mode/emulate-mobile-viewports.markdown [hunk 6]
```diff
@@ -1,122 +1,168 @@
...
+  2. **Specific Device**. Locks the Viewport to the exact viewport size of a specific device and emulates certain device characteristics.
 
-  ![mobile emulator on](imgs/emulator-on.png)
+## Responsive mode
 
-  Emulator off:
+We recommend using the **Responsive Mode** as your default work mode. Use it during active development of your site and app and resize the viewport often to create a freely responsive design that adapts to even unknown and future device types.
 
-  ![mobile emulator off](imgs/emulator-off.png)
+To get the most out of the Responsive Mode, turn on the [Media Queries Controls](./media-queries).
```
Comment (kaycebasques) [inline src/content/en/tools/chrome-devtools/iterate/device-mode/emulate-mobile-viewports.markdown]: 404

File: src/content/en/tools/chrome-devtools/iterate/device-mode/emulate-mobile-viewports.markdown [hunk 7]
```diff
@@ -1,122 +1,168 @@
...
+### Customize the viewport size
 
-  Portrait:
+Either drag the big resize handles on the viewport or click into the values in the menu bar for finer grained control.
 
-  ![portrait view](imgs/portrait.png)
+## Device-specific mode
 
-  Landscape:
+Use the **Device-specific Mode** when you're nearing the end of active development and want to perfect how your site looks like on specific mobiles (e.g. an certain iPhone or Nexus).
```
Comment (kaycebasques) [inline src/content/en/tools/chrome-devtools/iterate/device-mode/emulate-mobile-viewports.markdown]: s/an certain/a certain/

File: src/content/en/tools/chrome-devtools/iterate/device-mode/emulate-mobile-viewports.markdown [hunk 8]
```diff
@@ -1,122 +1,168 @@
...
+  </ul>
+  </div>
+  <div class="wf-devtools-flex-third">
+    <img src="imgs/device-mode-dotmenu.png" alt="Device Mode Settings">
+  </div>
+</div>
+
+Read on to learn more about the specific options.
+
+### User agent type (&gt; Chrome 50: Device type)
```
Comment (kaycebasques) [inline src/content/en/tools/chrome-devtools/iterate/device-mode/emulate-mobile-viewports.markdown]: This confused me. Are you saying that the name is different depending on what Chrome you're using? E.g. Chrome 49 or below, and Chrome 50 or above

50 lands in stable in only a month. I'd just focus on that and drop references to Chrome 49 and below

**Comment by kaycebasques** [comment]
@pbakaus take a look at my comments

**Comment by paulirish** [comment]
@crhym3 just gotta say, the webhook builder is AMAZING. thank you. :D

**Comment by crhym3** [comment]
@paulirish thanks, glad it turned out to be useful. there's still lots of room for improvement, will be working on that.

File: src/content/en/tools/chrome-devtools/iterate/device-mode/index.markdown (hunk 3 above)
Comment (pbakaus) [inline src/content/en/tools/chrome-devtools/iterate/device-mode/index.markdown]: Huh? That link is already gone :)

File: src/content/en/tools/chrome-devtools/iterate/device-mode/emulate-mobile-viewports.markdown (hunk 8 above)
Comment (pbakaus) [inline src/content/en/tools/chrome-devtools/iterate/device-mode/emulate-mobile-viewports.markdown]: I removed that line to get rid of the confusion. I want to make it a habit of documenting stable, as Canary changes too often. Feel free to fight me on this ;)

**Comment by pbakaus** [comment]
@kaycebasques thanks for the review, I fixed all issues!

On the annotation color: I propose we use blue throughout DevTools then (going forward, no need to change existing), since blue is the color of DevTools.

**Comment by kaycebasques** [comment]
@paulirish WDYT to proposal to use blue going forward?

**Comment by paulirish** [comment]
Blue sgtm

**Comment by petele** [comment]
Will add the redirects when I push to live.

**Pull request merged**